src/app/components/Sidebar/Sidebar.tsx

```diff
@@ -0,0 +1,28 @@
+import { classNames } from '../../utils/styles';
+import classes from './Sidebar.module.css';
+import { Settings, Info, Home, Favorites } from '../Icons';
+
+function Sidebar(): JSX.Element {
+  return (
+    <nav className={classNames(classes.container, classes.flex)}>
+      <section className={classNames(classes.section, classes.flex)}>
+        <a className={classes.selected}>
+          <Home selected={true} />
```

Comment:
Shortform for booleans

```suggestion
          <Home selected />
```

Reply:
ahhh cool, didn`t know that, thanks